app/assets/stylesheets/application.tailwind.css

```diff
@@ -21,3 +21,10 @@
 .icon.cursor-pointer:active {
   @apply scale-90 ease-in-out duration-300
 }
+
+
+/* Parent utility classes which activate children */
+
+.relationship-flex .relationship\:flex {
+  @apply !flex
+}
```

Comment:
This let's me apply "relationship-flex" to a parent element and any children that have "relationship:flex" get the flex activated. I'm copying the tailwind notation from "group-hover:..."